addOns/openapi/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@ All notable changes to this add-on will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## Unreleased
+### Changed
+- Add support for detecting data-driven nodes from schema
```

Comment:
Since this is all about DDN support the following items can be nested under this one.